extensions/systemd/index.js

```diff
@@ -22,6 +22,9 @@ class SystemdExtension extends Extension {
                 }
 
                 return false;
+            },
+            onUserSkip: ({instance}) => {
+                instance.config.set('process', 'local').save();
```

Comment:
last thing - do we want to print out a log message saying that we're reverting back to local process manager?

Reply:
Ooh yeah, good idea - `Using "local" process manager` (yellow)?

Reply:
maybe "Systemd setup skipped, reverting to local process manager", yellow sounds good color-wise though